tensorflow/contrib/pi_examples/README.md

```diff
@@ -0,0 +1,39 @@
+# TensorFlow Raspberry Pi Examples
+
+This folder contains examples of how to build applications for the Raspberry Pi using TensorFlow.
+
```

Comment:
Maybe:
"This folder contains an example of a Raspberry Pi application that uses TensorFlow", as there's only one example.